modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionFilters.spec.ts

```diff
@@ -0,0 +1,211 @@
+        });
+
+        await test.step('Add a field, so FDS has something to show', async () => {
+            await dataSetManagerApiHelpers.createDataSetField({
+                dataSetERC,
+                label_i18n: {en_US: fieldLabel},
+                name: 'rendererType',
+                type: 'string',
+            });
+        });
+
+        await test.step('Check that mandatory missing fields display an error message', async () => {
+			await expect(filtersPage.newFilterButton).toBeVisible();
+
+		    await filtersPage.newFilterButton.click();
+
+            const menuItem = filtersPage.page.getByRole('menuitem', {
+                name: 'Client Extension',
+            });
+
+            await expect(menuItem).toBeVisible();
+
+            await menuItem.click();
+
+            await filtersPage.saveAddFilterModal();
+
+            await expect(page.getByText('This field is required.')).toHaveCount(3);
+
+            await filtersPage.newClientExtensionFilterModal.nameInput.click();
+            await filtersPage.newClientExtensionFilterModal.nameInput.fill(filterLabel);
+            await filtersPage.saveAddFilterModal();
+
+            await expect(page.getByText('This field is required.')).toHaveCount(2);
+
+            await filtersPage.newClientExtensionFilterModal.filterBySelect.click();
+            await page.getByRole('option', {name: DATE_FIELD_NAME}).click();
+            await filtersPage.saveAddFilterModal();
+
+            await expect(page.getByText('This field is required.')).toHaveCount(1);
+
+            await filtersPage.newClientExtensionFilterModal.clientExtensionDropdown.click();
+            await page.getByRole('option', {name: clientExtensionName}).click();
+
+            await filtersPage.saveAddFilterModal();
+
+            await expect(page.getByText('This field is required.')).toHaveCount(0);
+
+            await filtersPage.cancelAddFilterModal();
+        });
+    });
+
+    test('Can create a Client Extension Filter in DSM', async({
+        dataSetManagerApiHelpers,
+        filtersPage,
+        page,
+    }) => {
+        const fieldLabel = getRandomString();
+        const filterLabel = getRandomString();
+
+        await test.step('Add a field, so FDS has something to show', async () => {
+            await dataSetManagerApiHelpers.createDataSetField({
+                dataSetERC,
+                label_i18n: {en_US: fieldLabel},
+                name: 'rendererType',
+                type: 'string',
+            });
+        });
+
+        await test.step('Create a client extension filter', async () => {
+            await filtersPage.createClientExtensionFilter({
+                filterBy: DATE_FIELD_NAME,
+                name: filterLabel,
+                clientExtension: clientExtensionName
+            });
+
+            await filtersPage.saveAddFilterModal();
+        });
+
+        await test.step('Check that the client extension filter is in the list', async () => {
+			await expect(
```

Comment:
missing SF? there's an issue with pw SF . You can use this code https://github.com/liferay-frontend/liferay-portal/pull/4308 to fix it

Reply:
Yep. I was waiting for this PR to be merged